code/datums/gamemode/factions/vox_shoal.dm

```diff
@@ -305,7 +305,7 @@ var/list/potential_bonus_items = list(
 	var/vox_raider_data = SSpersistence_misc.read_data(/datum/persistence_task/vox_raiders)
 	var/score_to_beat = vox_raider_data["best_score"]
 	var/best_team = vox_raider_data["winning_team"]
-	info = {"<h4>The shoal needs us to gather ressources. </h4>
+	info = {"<h4>The shoal needs us to gather resources. </h4>
```

Comment:
This is a 100% French typo that I made probably some years ago, I included it because it conflicts with the PR otherwise and I don't want to forget it